lab3/euler.cpp

```diff
@@ -0,0 +1,44 @@
+#include <fstream>
+
+int TIMENUMBERS  = 100000;
+double x_0 = 5;
+double v_0 = 0;
+double dt = 0.01;
+double omega = 0.5;
+
+void output(double *x, double *x1, double *v, double *v1, double *t)
```

Comment:
Cpp way would be to use const references to vars or dynamic containers (vectors) here instead of non-const pointers. Also how much modifications would it require to compare float and double precision results?